### PR TITLE
docs: `cluster_info` table

### PR DESCRIPTION
**Pull request by killme2008**
## What's Changed in this PR

Adds the docs for https://github.com/GreptimeTeam/greptimedb/pull/3832

## Checklist

- [x] Please ensure that the content in `summary.yml` matches the current document structure when you changed the document structure.
- [ ] This change requires follow-up update in localized docs.

### CODE REVIEW

File: docs/nightly/zh/user-guide/operations/region-migration.md [hunk 1]
```diff
@@ -32,6 +32,9 @@ on a.greptime_partition_id = b.region_id where a.table_name='migration_target' o
 1 row in set (0.01 sec)
 ```
 
+
+更多关于 `region_peers`  表的信息，请阅读 [Reference](/reference/sql/information-schema/cluster-info.md)。
```
Comment (nicecui) [inline docs/nightly/zh/user-guide/operations/region-migration.md:36]:
```suggestion
更多关于 `region_peers`  表的信息，请阅读 [REGION-PEERS](/reference/sql/information-schema/region-peers.md)。
```

File: docs/nightly/en/user-guide/operations/region-migration.md [hunk 2]
```diff
@@ -32,6 +32,9 @@ For example, have the following region distribution:
 1 row in set (0.01 sec)
 ```
 
+
+For more info about the `region_peers` table, please read the [reference](/reference/sql/information-schema/cluster-info.md).
```
Comment (nicecui) [inline docs/nightly/en/user-guide/operations/region-migration.md:36]:
```suggestion
For more info about the `region_peers` table, please read the [REGION-PEERS](/reference/sql/information-schema/region-peers.md).
```

**Review by nicecui — APPROVED** [review]

File: docs/nightly/zh/reference/sql/information-schema/cluster-info.md [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+* `peer_id`: 节点的服务器 ID。对于 standalone 来讲，该字段总是为 `0`，对于集群模式下的 frontend 该字段总为 `-1`。因为在这两种情况下，该字段没有实际含义。
+* `peer_type`: 节点类型, 分布式集群里可能是 `METASRV`、`FRONTEND` 或者 `DATANODE`。单机模式显示为 `STANDALONE`。
+* `peer_addr`: 节点的 gRPC 服务地址。对于单机部署，该字段总为空。
+* `version`: 节点的版本号，形如 `0.7.2` 的字符串。
+* `start_time`: 节点的启动时间。
+* `uptime`: 节点的持续运行时间，形如 `24h 10m 59s 150ms` 的字符串。
+* `active_time`: 距离节点上一次活跃（也就是发送心跳请求）过去的时间，形如 `24h 10m 59s 150ms` 的字符串。单机模式下该字段总为空。
+
+
+让我们尝试查询下这张表：
```
Comment (tisonkun) [inline docs/nightly/zh/reference/sql/information-schema/cluster-info.md:41]: @nicecui  IIRC generally it's not recommended to use `we`/`us` in documents. But I don't know if we have such guidance.

**Review by tisonkun — APPROVED** [review]
LGTM. Style comments inline.

File: docs/nightly/en/reference/sql/information-schema/cluster-info.md [hunk 4]
```diff
@@ -0,0 +1,70 @@
...
+
+* `peer_id`: the server id of the node. It's always `0` for standalone mode and `-1` for frontends because it doesn't make sense in such cases.
+* `peer_type`: the node type, `METASRV`,`FRONTEND`, or `DATANODE` for distributed clusters and `STANDALONE` for standalone deployments.
+* `peer_addr`: the GRPC server address of the node. It's always empty for standalone deployments.
+* `version`: The build version of the node, such as `0.7.2` etc.
+* `start_time`: The node start time.
+* `uptime`: The uptime of the node, in the format of duration string `24h 10m 59s 150ms`.
+* `active_time`: The duration string in the format of `24h 10m 59s 150ms` since the node's last active time(sending the heartbeats), it's always empty for standalone deployments.
+
+Let's query the table:
```
Comment (nicecui) [inline docs/nightly/en/reference/sql/information-schema/cluster-info.md:40]:
```suggestion
Query the table:
```

File: docs/nightly/zh/reference/sql/information-schema/cluster-info.md (hunk 3 above)
Comment (nicecui) [inline docs/nightly/zh/reference/sql/information-schema/cluster-info.md:41]:
```suggestion
尝试查询下这张表：
```

**Pull request merged**